libr/main/rabin2.c

```diff
@@ -893,6 +893,12 @@ R_API int r_main_rabin2(int argc, const char **argv) {
 		return 1;
 	}
 	file = argv[opt.ind];
+
+	if (file && file[0] == '\0') {
```

Comment:
why && and not || ? I think it should be || and in that case use R_STR_ISEMPTY

Reply:
because then it means that the if will handle the case where the user did not insert a file at all, where I would expect the program to print the print help and not complain that the path string is an empty string. the case of no file at all is handled in the if block right under my patch.

btw, I should probably call r_core_fini(&core) as in other places, before returning, noted.

Reply:
or maybe this _should_ be treated as when the user passes no file at all?

Reply:
Right, no then I guess it is fine as it is. And yes, you should probably call `r_core_fini` as you said. Thanks!